perceval/converters/abstract_converter.py

```diff
@@ -52,8 +52,7 @@ class AGateConverter(ABC):
     def __init__(self, backend_name: str = "SLOS", source: Source = Source()):
         self._converted_processor = None
         self._input_list = None  # input state in list
-        self._cnot_idx = 0  # counter for CNOTS in circuit
-        self._source = source
+        self._noise_model = NoiseModel()
```

Comment:
You're replacing `Source` with `NoiseModel` which is cool, but please go to the end of it:
* deprecated the source usage
* have a `noise` parameter in the `__init__` of all converters.